test/bazel-karma-local-config.js

```diff
@@ -3,12 +3,12 @@
  * want to launch any browser and just enable manual browser debugging.
  */
 
-const bazelKarma = require('@bazel/karma');
+const bazelKarma = require('@bazel/concatjs');
```

Comment:
I find it really strange that they changed these rules to live under a package named for the bundler 😕

Reply:
Yeah, my guess is that its about consolidation overall: https://github.com/bazelbuild/rules_nodejs/wiki#introducing-bazelconcatjs